commons/src/test/java/io/hyscale/commons/config/TestSetupConfig.java

```diff
@@ -0,0 +1,101 @@
+/**
+ * Copyright 2019 Pramati Prism, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.hyscale.commons.config;
+
+import io.hyscale.commons.constants.ToolConstants;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class TestSetupConfig {
+    private static String appName;
+    private static String svcName;
+    private static StringBuilder APPS_DIR;
+    private static StringBuilder SERVICE_DIR;
```

Comment:
Since you want the apps_dir & service_dir to be a global variable . Make them available as a String rather than a StringBuilder